app/scripts/directives/createSecret.js

```diff
@@ -174,6 +187,11 @@ angular.module("openshiftConsole")
                 secret.stringData.WebHookSecretKey = data.webhookSecretKey;
               }
               break;
+            case "Generic":
+              if (data.genericKeyValues.data) {
+                secret.stringData = data.genericKeyValues.data;
```

Comment:
We'll need a follow on to handle binary content for secrets, but this is fine for now.

Reply:
Ok, filed issue #2896